app/models/manageiq/providers/cloud_manager/template.rb

```diff
@@ -111,6 +150,25 @@ def delete_image_queue(userid)
     MiqTask.generic_action_with_callback(task_opts, queue_opts)
   end
 
+  def import_image_queue(userid)
```

Comment:
What is a template instance method buying us if we aren't importing this image?

Reply:
The method queues an event, which must be processed in the provider. I made it similar to how e.g. **create_image_queue, update_image_queue, delete_image_queue** do that.

Reply:
create_image_queue doesn't have an instance method though, if you have an instance it is already create and I think the same rule applies here.

The only thing I could see making sense on an image instance would be export

Reply:
Oh, I get what you meant. I misunderstood. Yeah, the method doesn't have to be static.